lib/simd/Grid_generic_types.h

```diff
@@ -26,7 +26,7 @@ Author: Antonin Portelli <antonin.portelli@me.com>
     See the full license in the file "LICENSE" in the top level distribution directory
     *************************************************************************************/
     /*  END LEGAL */
-#define GEN_SIMD_WIDTH 16
```

Comment:
@nmeyer-ur it is ok that you deleted from here. But don't you need somewhere this definition? For example in the NEON file?